project-base/storefront/config/staticRewritePaths.js

```diff
@@ -25,7 +25,7 @@ const STATIC_REWRITE_PATHS = {
         '/cookie-consent': '/cookie-consent',
         '/abandoned-cart/:cartUuid': '/abandoned-cart/:cartUuid',
         '/grapesjs-template': '/grapesjs-template',
-        '/products-comparison': '/products-comparison',
+        '/product-comparison': '/product-comparison',
```

Comment:
is this change necessary? I think that products - is even better - bcs we compare more than one product. This change only have more cons then pluses.

Reply:
Answered here:
https://github.com/shopsys/shopsys/pull/2717#discussion_r1293280420